anker/src/state.rs

```diff
@@ -532,19 +536,9 @@ impl Anker {
             accounts.config_key.key,
         )?;
         check_wormhole_account(
-            "custody key",
-            &wormhole_transfer_args.custody_key,
-            accounts.custody_key.key,
-        )?;
-        check_wormhole_account(
```

Comment:
Very good catch! Fixed. I think what happened here is that I removed the custody key and custody signer key, because they are only used for native tokens, not wrapped tokens like UST, but I was a bit too eager and also removed this check in between that I shouldn’t have removed.